task_status_fsm.dot

```diff
@@ -0,0 +1,49 @@
+digraph task_status_fsm {
```

Comment:
:+1: :100: :+1: 

but can you put in in docs or something? or even next to the MarathonTaskStatus...